modules/user/user.service.js

```diff
@@ -0,0 +1,46 @@
+const userRepository = require("./user.repository");
+
+class UserService {
+    async readAllUsers() {
+        return userRepository.readAllUsers();
+    }
+    async readOneUser(id) {
+        const user = await userRepository.readUserById(id);
+        return user;
+    }
+
+    async createUser(newUser) {
+        const maxUsersId = await userRepository.findMaxUserId();
+        newUser.id = maxUsersId + 1;
+        const savedUser = await userRepository.saveNewUser(newUser);
+        return savedUser;
+    }
+
+    async updateUser(updatedUser) {
+        const savedUser = await userRepository.updateUser(updatedUser);
+        return savedUser;
+    }
+
+    async deleteUser(id) {
+        const deletedUser = await userRepository.deleteUser(id);
+        return deletedUser;
+    }
+
+    async getUser(request) {
```

Comment:
Request shouldn't be mentioned in service at all. Remove request from this function argument and leave here only expected userBody.